person.rb

```diff
@@ -1,14 +1,19 @@
-class Person
+class Person < Nameable
```

Comment:
- [ ] When I tested your app I got error due to missing files. For this inheritance to work you need to import the `nameable` file here.

  ```
  require_relative './nameable.rb'
  ```